### ISSUE TITLE
Node.js Technical Steering Committee (TSC) Meeting 2022-11-16

### ISSUE THREAD

**Issue by mhdawson** [nodejs/TSC#1307]
## Time

**UTC Wed 16-Nov-2022 13:00 (01:00 PM)**:

| Timezone      | Date/Time             |
|---------------|-----------------------|
| US / Pacific  | Wed 16-Nov-2022 05:00 (05:00 AM) |
| US / Mountain | Wed 16-Nov-2022 06:00 (06:00 AM) |
| US / Central  | Wed 16-Nov-2022 07:00 (07:00 AM) |
| US / Eastern  | Wed 16-Nov-2022 08:00 (08:00 AM) |
| EU / Western  | Wed 16-Nov-2022 13:00 (01:00 PM) |
| EU / Central  | Wed 16-Nov-2022 14:00 (02:00 PM) |
| EU / Eastern  | Wed 16-Nov-2022 15:00 (03:00 PM) |
| Moscow        | Wed 16-Nov-2022 16:00 (04:00 PM) |
| Chennai       | Wed 16-Nov-2022 18:30 (06:30 PM) |
| Hangzhou      | Wed 16-Nov-2022 21:00 (09:00 PM) |
| Tokyo         | Wed 16-Nov-2022 22:00 (10:00 PM) |
| Sydney        | Thu 17-Nov-2022 00:00 (12:00 AM) |

Or in your local time:
* https://www.timeanddate.com/worldclock/fixedtime.html?msg=Node.js+Foundation+Technical%20Steering%20Committee%20(TSC)+Meeting+2022-11-16&iso=20221116T1300
* or https://www.wolframalpha.com/input/?i=01PM+UTC%2C+Nov+16%2C+2022+in+local+time

## Links

* Minutes Google Doc: <https://docs.google.com/document/d/1Ed0cQ8Sx9XWFAIsbw_hZbEolHpsb-yOuoz4Uplm6Vvw/edit>

## Agenda

Extracted from **tsc-agenda** labelled issues and pull requests from the **nodejs org** prior to the meeting.

### nodejs/node

* async_hooks: add migration note [#45335](https://github.com/nodejs/node/pull/45335)
* doc: add Node.js Threat Model [#45223](https://github.com/nodejs/node/pull/45223)

### nodejs/TSC

* New meeting times [#1306](https://github.com/nodejs/TSC/issues/1306)

## Invited

* Antoine du Hamel @aduh95 (TSC)
* Anatoli Papirovski @apapirovski (TSC)
* Beth Griggs @BethGriggs (TSC)
* Ruben Bridgewater @BridgeAR (TSC)
* Сковорода Никита Андреевич @ChALkeR (TSC)
* Colin Ihrig @cjihrig (TSC)
* Danielle Adams @danielleadams (TSC)
* Franziska Hinkelmann @fhinkel (TSC)
* Geoffrey Booth @GeoffreyBooth (TSC)
* Gireesh Punathil @gireeshpunathil (TSC)
* James Snell @jasnell (TSC)
* Joyee Cheung @joyeecheung (TSC)
* Chengzhong Wu @legendecas (TSC)
* Matteo Collina @mcollina (TSC)
* Michael Dawson @mhdawson (TSC)
* Darshan Sen @RaisinTen (TSC)
* Richard Lau @richardlau (TSC)
* Robert Nagy @ronag (TSC)
* Michaël Zasso @targos (TSC)
* Tobias Nießen @tniessen (TSC)
* Rich Trott @Trott (TSC)

## Notes

The agenda comes from issues labelled with `tsc-agenda` across **all of the repositories in the nodejs org**. Please label any additional issues that should be on the agenda before the meeting starts.

## Joining the meeting

Zoom link: <https://zoom.us/j/611357642>
Regular password

## Public participation

We stream our conference call straight to YouTube so anyone can listen to it live, it should start playing at **<https://www.youtube.com/c/nodejs+foundation/live>** when we turn it on. There's usually a short cat-herding time at the start of the meeting and then occasionally we have some quick private business to attend to before we can start recording & streaming. So be patient and it should show up.

---

**Invitees**

Please use the following emoji reactions in this post to indicate your
availability.

* :+1: - Attending
* :-1: - Not attending
* :confused: - Not sure yet

**Comment by mcollina** [comment]
I'm traveling to London and I won't be able to attend or chair.

**Comment by GeoffreyBooth** [comment]
I can't attend this time of day. Can we postpone "async_hooks: add migration note #45335" to next time?

Not sure if the other items need a meeting?

**Comment by gireeshpunathil** [comment]
I can chair, in case we decide to go ahead.

**Comment by Trott** [comment]
New meeting times https://github.com/nodejs/TSC/issues/1306 can be taken off the agenda. (I'll do that now.)

**Comment by GeoffreyBooth** [comment]
I took “async_hooks: add migration note [#45335](https://github.com/nodejs/node/pull/45335)” off the agenda as I think both I and @mcollina need to be present for that one (ideally @jasnell and @cjihrig as well). We can leave the label on the PR so that it gets added to the agenda for next week.

Is there a question for “doc: add Node.js Threat Model https://github.com/nodejs/node/pull/45223”? If not I think we can cancel.

**Comment by mhdawson** [comment]
There was no specific question on Threat Model https://github.com/nodejs/node/pull/45223, just important that TSC members are aware. It looks like it is getting reviews/LGTMs so I don't think we have anything to discuss in the meeting this week. I'll also remove the agend Item.

I'm cancel the meeting as we no longer have any agenda items and don't see any new ones being suggested. Please let me know if that was not the right thing to do.
